### PR TITLE
Test to check that bar graph makes requests with size 1 (limit=1).

### PR DESCRIPTION
**Pull request by shefalijoshi**
Closes https://github.com/nasa/openmct/pull/7811

### Describe your changes:
Include Graphs in default plugins installed.
Test to check that bar graph makes requests with size 1 (limit=1)

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [ ] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [ ] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: tests/e2e/yamcs/barGraph.e2e.spec.mjs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+    // Create the Bar Graph
+    barGraph = await createDomainObjectWithDefaults(page, { type: 'Graph' });
+  });
+
+  test('Requests a single historical datum', async ({ page }) => {
+    //Expand the myproject folder (/myproject)
+    const myProjectTreeItem = page.locator('.c-tree__item').filter({ hasText: 'myproject'});
+    const firstMyProjectTriangle = myProjectTreeItem.first().locator('span.c-disclosure-triangle');
```
Comment (unlikelyzero) [inline tests/e2e/yamcs/barGraph.e2e.spec.mjs:47]: We've moved away from the .locator pattern. Let's make sure to the the aria `Expand my project` logic

**Review by unlikelyzero — CHANGES_REQUESTED** [review]
let's modify the tests a bit

File: tests/e2e/yamcs/barGraph.e2e.spec.mjs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    // Open a browser, navigate to the main page, and wait until all networkevents to resolve
+    await page.goto('./', { waitUntil: 'domcontentloaded' });
+
+    // Create the Bar Graph
+    barGraph = await createDomainObjectWithDefaults(page, { type: 'Graph' });
+    // Enter edit mode for the overlay plot
+    await page.getByLabel('Edit Object').click();
+  });
+
+  test('Requests a single historical datum', async ({ page }) => {
```
Comment (akhenry) [inline tests/e2e/yamcs/barGraph.e2e.spec.mjs:45]: Where's the assertion in this test?

**Review by akhenry — CHANGES_REQUESTED** [review]

File: tests/e2e/yamcs/barGraph.e2e.spec.mjs (hunk 2 above)
Comment (shefalijoshi) [reply tests/e2e/yamcs/barGraph.e2e.spec.mjs:45]: @akhenry https://github.com/akhenry/openmct-yamcs/pull/470/files/f62b4c54baa53ffda462a91b7814032216ae2b85#diff-c9a10b8751a2a70edf45ed5991bb3728276e81ec7ca795d39724f88265f055b0R51

The assertion is that the `historicalGet` with `limit=1` resolves

File: package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "lint:fix": "eslint src example --fix",
     "build:dist": "webpack --config ./.webpack/webpack.prod.mjs",
     "build:example": "npm install openmct@unstable --no-save",
-    "build:example:master": "npm install nasa/openmct --no-save",
+    "build:example:master": "npm install nasa/openmct#bar-graph-historical-requests --no-save",
```
Comment (unlikelyzero) [inline package.json:23]: here

File: tests/e2e/yamcsAppActions.mjs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ * away the underlying functionality of the application. For more about the App Action pattern, see /e2e/README.md)
+ *
+ * For example, if two functions are nearly identical in
+ * timer.e2e.spec.js and notebook.e2e.spec.js, that function should be generalized and moved into this file.
+ */
+
+/**
+ * Search for telemetry and link it to an object. objectName should come from the domainObject.name function.
+ * @param {import('@playwright/test').Page} page 
+ * @param {string} telemetryText
```
Comment (unlikelyzero) [inline tests/e2e/yamcsAppActions.mjs:35]: rename to parameterNameText

**Review by unlikelyzero — APPROVED** [review]
Need another reviewer

File: tests/README.md [hunk 5]
```diff
@@ -4,6 +4,14 @@ This project is using Open MCT's e2e-as-a-dependency model. To learn more, pleas
 
 ## How to Run Locally
 
+** Note: if you are running the tests to match a specific branch of openmct, you will need to run `npm run build:example:currentbranch` **
+
+### Makefile
+
+1. `make test-all`
```
Comment (ozyx) [inline tests/README.md:11]: ahhhhh

File: tests/e2e/yamcsAppActions.mjs (hunk 4 above)
Comment (ozyx) [reply tests/e2e/yamcsAppActions.mjs:35]: rename to `parameterName`

File: tests/e2e/yamcs/barGraph.e2e.spec.mjs [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    // Enter edit mode for the overlay plot
+    await searchAndLinkTelemetryToObject(page, 'Magnetometer', barGraph.name);
+  });
+
+  test('Requests a single historical datum', async ({ page }) => {
+
+    //http://localhost:9000/yamcs-proxy/api/archive/myproject/parameters/myproject/Magnetometer?start=2024-09-25T14%3A08%3A46.244Z&stop=2024-09-25T14%3A38%3A46.245Z&limit=1&order=desc
+    historicalGet = page.waitForRequest(/.*\/api\/.*\/parameters.*limit=1&order=desc$/);
+
+    await page.goto(barGraph.url, { waitUntil: 'networkidle' });
```
Comment (ozyx) [inline tests/e2e/yamcs/barGraph.e2e.spec.mjs:51]: prefer `domcontentloaded`  here. the wait for the parameters request should cover it, no?

**Review by ozyx — DISMISSED** [review]
mostly LGTM just have a couple requests

File: tests/e2e/yamcs/barGraph.e2e.spec.mjs (hunk 6 above)
Comment (unlikelyzero) [reply tests/e2e/yamcs/barGraph.e2e.spec.mjs:51]: I could see it either way. I don't like how this one fails in 30s on the promise. We need a better assertion

**Pull request merged**